arches/app/media/js/views/components/search/map-filter.js

```diff
@@ -274,6 +275,39 @@ define([
                 }
             }, this);
 
+            this.filterByFeatureGeom = function(resourceid, geoms) {
+                let feature = geoms[0].geom.features[0];
+                if (feature.geometry.type == 'Point') { self.buffer(100); }
+                if (feature.id) {
+                    self.searchGeometryFeature({
+                        "featureid": feature.id,
+                        "resourceid": resourceid,
+                        "buffer": {
+                            "width": this.buffer(),
+                            "unit": this.bufferUnit()
+                        },
+                        "inverted": this.filter.inverted()
+                    });
+                } else {
+                    const tolerance = 0.1; // Degree of Simplification: Lower numbers are less simplified, preserving more detail
+                    const highQuality = true; // Set to true for a slower but higher quality simplification
+                    turf.simplify(feature.geometry, {tolerance: tolerance, highQuality: highQuality, mutate: true});
+                }
```

Comment:
Simplifying the geometries used in the search filter seems out of scope for the issue associated with this PR ( #10816). It may be something that we should do to shorten the search URL, but changing a geometry without a user's awareness seems like an important enough change that it should be addressed in another issue/PR.

Reply:
@chiatt, @whatisgalen - I'm just trying to finish up another time-critical task and then I'll delve in further, but it seems like it would be a good thing to externalize the the functionality that provides the search geometry into a utility class (kind of like what we did for the map popups) so that custom logic (eg - whether you want to simplify geometries, or getting the geometry by another method, etc, etc) could be overridden by the developer without delving into the core Arches javascript files which are more tightly integrated into the core arches components (cards, views, widgets, etc).

Reply:
> @chiatt, @whatisgalen - I'm just trying to finish up another time-critical task and then I'll delve in further, but it seems like it would be a good thing to externalize the the functionality that provides the search geometry into a utility class (kind of like what we did for the map popups) so that custom logic (eg - whether you want to simplify geometries, or getting the geometry by another method, etc, etc) could be overridden by the developer without delving into the core Arches javascript files which are more tightly integrated into the core arches components (cards, views, widgets, etc).

@bferguso thanks for this great feedback. I've incorporated your suggestion into creating a method in the map popup provider to handle the popup feature en route to sending to the mapCard's method. Unless a developer has overridden `viewmodels/map.js`, every popup feature should have direct access to the `mapCard` i.e. Map Filter search filter.

Reply:
> Simplifying the geometries used in the search filter seems out of scope for the issue associated with this PR ( #10816). It may be something that we should do to shorten the search URL, but changing a geometry without a user's awareness seems like an important enough change that it should be addressed in another issue/PR.

removed in latest